src/gui/CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
 project(gui)
 find_package(Qt5 REQUIRED COMPONENTS Widgets Svg Qml Quick QuickControls2 Xml Network)
+
+if(QUICK_COMPILER)
+        find_package(Qt5QuickCompiler)
+        set_package_properties(Qt5QuickCompiler PROPERTIES
+                DESCRIPTION "Compile QML at build time"
+                TYPE OPTIONAL
```

Comment:
should not this be a `REQUIRED` as the user needs to explicitly define `QUICK_COMPILER` to run into it so I would say they want to make sure they use it

Reply:
True, now set to required